skll/data/featureset.py

```diff
@@ -24,6 +24,8 @@ class FeatureSet(object):
     """
     Encapsulation of all of the features, values, and metadata about a given
     set of data.
+    warning:: FeatureSets can only be equal if the order of the instances is
```

Comment:
Almost!  You want this to be:

```rst
Encapsulation of all of the features, values, and metadata about a given
set of data.

.. warning:: 
    FeatureSets can only be equal if the order of the instances is identical 
    because these are stored as lists/arrays.
```